golem_messages/factories/tasks.py

```diff
@@ -99,7 +159,18 @@ def task_id(
         if ttc.compute_task_def is None:
             return
 
-        ttc.compute_task_def['task_id'] = extracted or helpers.fake_golem_uuid(  # noqa pylint: disable=unsupported-assignment-operation
+        _task_id = None
+        with suppress(AttributeError):
```

Comment:
```python
>>> import this
[...]
"There should be one-- and preferably only one --obvious way to do it."
```
I'm not convinced to this suppress thing :). It doesn't add anything to context.

Reply:
hm... I would say is more succinct and better expresses intention of doing nothing (ignoring) in case particular exceptions shows up